crates/builder-api/src/v0_1/data_source.rs

```diff
@@ -70,4 +70,9 @@ where
         &self,
         txns: Vec<<I as NodeType>::Transaction>,
     ) -> Result<Vec<Commitment<<I as NodeType>::Transaction>>, BuildError>;
+
+    async fn get_tx_status(
```

Comment:
Sorry for nitpicking, but I think for function names we don't want to include `get`. We add `get` only for endpoint names.

Reply:
Updated to `txn_status`.